polkadot/runtime/parachains/src/paras_inherent/tests.rs

```diff
@@ -2980,6 +2986,272 @@ mod sanitizers {
 			}
 		}
 
+		// Para 1 scheduled on cores 0, 1 and 2. Three candidates are supplied but their relay
+		// parents look like this: 3, 2, 3.
+		fn get_test_data_for_relay_parent_ordering(core_index_enabled: bool) -> TestData {
```

Comment:
Do we have test coverage for rp being correctly ordered: `2,3,3` for example ?

Reply:
good suggestion. added that